api/src/main/java/org/apache/gravitino/file/FilesetChange.java

```diff
@@ -74,6 +74,7 @@ static FilesetChange removeProperty(String property) {
    *
    * @return The fileset change.
    */
+  @Deprecated
```

Comment:
Plz also add the Deprecated reason to the comment and give the new usage recommendations

Reply:
BTW, you should change the comment like this:
```java
 /**
   * Creates a new fileset change to remove comment from the fileset.
   *
   * @return The fileset change.
   * @deprecated Use {@link #updateComment(String)} with null value as the argument instead.
   */
```